libs/env/src/env.ts

```diff
@@ -102,6 +102,11 @@ const envDefinitions = {
     def: "https://filecoin.coin.ledger.com",
     desc: "Filecoin API url",
   },
+  API_KADENA_ENDPOINT: {
+    parser: stringParser,
+    def: "https://estats.testnet.chainweb.com",
+    desc: "Kadena Indexer API url",
+  },
```

Comment:
This might need to change that they have new way to do this now, I am taking example from coin-near/config.ts